frontend/messages/en.json

```
@@ -3054,6 +3054,12 @@
 	"metricDefinitionsDescription": "Define custom metrics to track and measure performance indicators",
 	"metricInstancesDescription": "Track specific metric values for objects across your organization",
 	"dashboardsDescription": "Custom dashboards to visualize metrics and key performance indicators",
+	"libraryFilteringLabels": "Library labels",
+	"requirementAssessmentDifferentBranch": "Requirement assessment belongs to another branch than the audit",
```

Biome (2.1.2)

[error] The key libraryFilteringLabels was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)